pkg/datarepair/checker/online.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package checker
+
+import (
+	"context"
+	"time"
+
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+)
+
+// ReliableCache caches the reliable nodes for the specified staleness duration
+// and updates automatically from overlay.
+type ReliableCache struct {
```

Comment:
Can this be "ReliabilityCache" instead? "ReliableCache" implies to me "cache that is reliable", and that caused me some confusion in reviewing here

Reply:
renamed